README.md

```diff
@@ -443,13 +448,7 @@ Qemu provides support for using BrlAPI to display braille output on a
 real or fake device.
 
 ``` {.bash}
-quickemu --vm ubuntu-impish.conf --accessibility braille --display sdl
-```
-
-or even
-
-``` {.bash}
-quickemu --vm ubuntu-impish.conf --acc brl --display sdl
+quickemu --vm ubuntu-impish.conf --braille --display sdl
```

Comment:
and this change needs to make it in too.

I'll pop in a separate PR for just the Readme bits to leave you the option